packages/zowe-explorer/src/trees/dataset/DatasetTree.ts

```diff
@@ -112,16 +112,26 @@ export class DatasetTree extends ZoweTreeProvider<IZoweDatasetTreeNode> implemen
         recursiveCall?: boolean
     ): Promise<void> {
         const destinationInfo = FsAbstractUtils.getInfoForUri(destUri, Profiles.getInstance());
-        let dsname = destUri.path.substring(destinationInfo.slashAfterProfilePos);
+        const sourceInfo = FsAbstractUtils.getInfoForUri(sourceUri, Profiles.getInstance());
+        let dsname = destUri.path.substring(destinationInfo.slashAfterProfilePos + 1);
         if (SharedContext.isPds(sourceNode)) {
             if (!DatasetFSProvider.instance.exists(destUri)) {
-                // create a PDS on remote
+                // find source PDS attributes and make destination PDS have same attributes
+                const sourceAttributesResponse = await ZoweExplorerApiRegister.getMvsApi(sourceInfo.profile).dataSet(dsname, {
+                    attributes: true,
+                    responseTimeout: sourceInfo.profile?.profile?.responseTimeout,
+                });
+                const { dsname: dsnameSource, ...rest } = sourceAttributesResponse.apiResponse.items[0];
+
+                // create a PDS on remote with the same attributes as source PDS
+                const transformedAttrs = (zosfiles.Copy as any).dataSetCrossLPAR({}, rest);
+                const isBinary = transformedAttrs.recfm === "U" && transformedAttrs.lrecl === 0;
                 try {
                     dsname = sourceNode.getLabel() as string;
                     await ZoweExplorerApiRegister.getMvsApi(destinationInfo.profile).createDataSet(
-                        zosfiles.CreateDataSetTypeEnum.DATA_SET_PARTITIONED,
+                        isBinary ? zosfiles.CreateDataSetTypeEnum.DATA_SET_BINARY : zosfiles.CreateDataSetTypeEnum.DATA_SET_PARTITIONED,
                         dsname,
-                        {}
+                        transformedAttrs
                     );
```

Comment:
Agree with Trae's suggestion here - I think we want to keep `generateDatasetOptions`.

Also the `isBinary` check may be unnecessary. We could use a blank data set template if all the attributes are specified on the `transformedAttrs` object.

Something like this should work:
```suggestion
                // create a PDS on remote with the same attributes as source PDS
                const transformedAttrs = (zosfiles.Copy as any).generateDatasetOptions({}, rest);

                try {
                    dsname = sourceNode.getLabel() as string;
                    await ZoweExplorerApiRegister.getMvsApi(destinationInfo.profile).createDataSet(
                        zosfiles.CreateDataSetTypeEnum.DATA_SET_BLANK,
                        dsname,
                        transformedAttrs
                    );
```